src/masternode-payments.cpp

```diff
@@ -777,16 +777,10 @@ std::string CMasternodePayments::ToString() const
     return info.str();
 }
 
-bool CMasternodePayments::IsEnoughData(int nMnCount) {
-    if(GetBlockCount() > nMnCount * nStorageCoeff && GetBlockCount() > nMinBlocksToStore)
```

Comment:
I don't fully understand the the logic behind how this function is used but doesn't the block count also play a role ?  I'm surprised to see this condition disappear.